envs/example/ci-full-ubuntu/heat_stack.yml

```diff
@@ -0,0 +1,240 @@
+          protocol: tcp
+          port_range_min: 8042
+          port_range_max: 8042
+        - remote_ip_prefix: 0.0.0.0/0
+          protocol: tcp
+          port_range_min: 6080
+          port_range_max: 6080
+        - remote_ip_prefix: 0.0.0.0/0
+          protocol: tcp
+          port_range_min: 8774
+          port_range_max: 8774
+        - remote_ip_prefix: 0.0.0.0/0
+          protocol: tcp
+          port_range_min: 8776
+          port_range_max: 8776
+        - remote_ip_prefix: 0.0.0.0/0
+          protocol: tcp
+          port_range_min: 8777
+          port_range_max: 8777
+        - remote_ip_prefix: 0.0.0.0/0
+          protocol: tcp
+          port_range_min: 9393
+          port_range_max: 9393
+        - remote_ip_prefix: 0.0.0.0/0
+          protocol: tcp
+          port_range_min: 9292
+          port_range_max: 9292
+        - remote_ip_prefix: 0.0.0.0/0
+          protocol: tcp
+          port_range_min: 9696
+          port_range_max: 9696
+        - remote_ip_prefix: 0.0.0.0/0
+          protocol: icmp
+        - ethertype: 'IPv4'
+          remote_mode: 'remote_group_id'
+          protocol: tcp
+        - ethertype: 'IPv4'
+          remote_mode: 'remote_group_id'
+          protocol: udp
+        - ethertype: 'IPv4'
+          remote_mode: 'remote_group_id'
+          protocol: igmp
+        - ethertype: IPv4
+          remote_ip_prefix: 192.168.0.0/22
+          protocol: igmp
+
+  ssh_keypair:
+    type: OS::Nova::KeyPair
+    properties:
+      save_private_key: true
+      name: { get_param: key_name }
+
+  controller-0:
+    type: OS::Nova::Server
+    properties:
+      name: { get_param: controller-0_name }
+      image: { get_param: image }
+      flavor: { get_param: flavor }
+      key_name: { get_resource: ssh_keypair }
+      user_data_format: RAW
+      security_groups: [{ get_resource: server_security_group }]
+      networks:
+        - network: { get_param: network }
+      availability_zone: { get_param: az }
+      # placeholder line
+
+  controller-1:
+    type: OS::Nova::Server
+    properties:
+      name: { get_param: controller-1_name }
+      image: { get_param: image }
+      flavor: { get_param: flavor }
+      key_name: { get_resource: ssh_keypair }
+      user_data_format: RAW
+      security_groups: [{ get_resource: server_security_group }]
+      networks:
+        - network: { get_param: network }
+      availability_zone: { get_param: az }
+      # placeholder line
+
+  compute-0:
+    type: OS::Nova::Server
+    properties:
+      name: { get_param: compute-0_name }
+      image: { get_param: image }
+      flavor: { get_param: flavor }
+      key_name: { get_resource: ssh_keypair }
+      user_data_format: RAW
+      security_groups: [{ get_resource: server_security_group }]
+      networks:
+        - network: { get_param: network }
+      availability_zone: { get_param: az }
+      # placeholder line
+
+
+### Floating IP resources and associations for servers
```

Comment:
We don't want to assign FIPs here because we only have a limited amount of FIPs. These are just here for developers. It's just a quick way for them to assign FIP. They can just uncomment a few lines, when they do need a FIP.